doc/source/f2py/usage.rst

```diff
@@ -161,12 +161,29 @@ distinguished by the usage of ``-c`` and ``-h`` switches:
      for ``-l``.
  
    ``link-<resource>``
- 
      Link extension module with <resource> as defined by
      ``numpy_distutils/system_info.py``. E.g. to link with optimized
      LAPACK libraries (vecLib on MacOSX, ATLAS elsewhere), use
      ``--link-lapack_opt``. See also ``--help-link`` switch.
- 
+
+   .. note:: The ``f2py -c`` option must be applied either to an existing ``.pyf`` file (plus the source/object/library files) or one must specify the ``-m <modulename>`` option (plus the sources/object/library files). ``f2py -c`` is not meant to compile the generated C/API extension modules directly (although it is likely able to do that with proper include directory flags and the source/object file inputs). Use one of the following options:
```

Comment:
The sentence starting with `f2py -c is not meant ...` does not contain useful information and should be removed.